tools/harness-thci/OpenThread.py

```diff
@@ -551,6 +551,17 @@ def __setAddressfilterMode(self, mode):
         except Exception as e:
             ModuleHelper.WriteIntoDebugLogger('__setAddressFilterMode() Error: ' + str(e))
 
+    def __skipSeqNoIncrease(self):
+        """skip sequence number increase when recovering BBR Dataset from Network Data
+
+        Returns:
+            True: successful to set the behavior.
+            False: fail to set the behavior.
+        """
+        print('call __skipSeqNoIncrease()')
+        cmd = 'bbr skipseqnoinc'
```

Comment:
nit: May consistently use either `seqno` or `seqnum` in this PR?

Reply:
You mean instead of `skipseqnoinc`? I'm not quite happy with this name, but couldn't find other that explains the purpose. Notice that we already have `bbr config [seqno <seqno>] [delay <delay>] [timeout <timeout>]` and might be confusing to mix them.

Reply:
@edmont, I belive @librasungirl is commenting on the inconsistency between the use of `seqno` and `seqnum`. For example, this PR introduces both:
- `ConfigSkipSeqNumIncrease()`
- `skipseqnoinc`

Thoughts on picking `seqno` or `seqnum` and consistently using it everywhere?

Reply:
Understood. Done.